ocs_ci/ocs/constants.py

```diff
@@ -68,9 +68,13 @@
 ROOK_CSI_CEPHFS_DIR = os.path.join(ROOK_EXAMPLES_DIR, "csi", "cephfs")
 CLEANUP_YAML = "cleanup.yaml.j2"
 MANIFESTS_DIR = "manifests"
+
+
```

Comment:
remove blank line here

Reply:
Sometimes there were two empty lines between sections, sometime one, I deleted it on every place were we had 2 and we should now follow only one. Thanks